src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/DescriptorReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,120 @@
+Feature: Validate the reference of descriptors when creating resources
+
+        Background
+            Given this student exists:
+                  | studentUniqueId | birthDate  | firstName | lastSurname |
+                  | 604824          | 2010-01-13 | Traci     | Mathews     |
+
+
+        @ignore
+        Scenario: User CAN'T create a resource when descriptor doesn't exist
+             When client sends a POST request to "/ed-fi/localEducationAgencies" with body
+                  """
+                  {
+                        "localEducationAgencyId": 25590100,
+                        "nameOfInstitution": "Grand+Institution+Test",
+                        "categories": [
+                            {
+                              "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#Fake"
+                            }
+                        ],
+                        "localEducationAgencyCategoryDescriptor": "uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor#Fake"
+                  }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                  {"detail":"Data validation failed. See 'validationErrors' for details.","type":"urn:ed-fi:api:bad-request:data","title":"Data Validation Failed","status":400,"correlationId":null,"validationErrors":{"$.localEducationAgencyCategoryDescriptor":["LocalEducationAgencyCategoryDescriptor value 'uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor#Fake' does not exist."],"$.categories[0].educationOrganizationCategoryDescriptor":["EducationOrganizationCategoryDescriptor value 'uri://ed-fi.org/EducationOrganizationCategoryDescriptor#Fake' does not exist."]}}
+                  """
+
+        @ignore
+        Scenario: User can POST a resource when descriptor exists
+             When client sends a POST request to "/ed-fi/localEducationAgencies" with body
```

Comment:
Can we replace this to
`When a XXXXXXX request is made to "XXXXXXX"`

Reply:
Done